[ 2022-04-27T11:22:18     INFO ] logging to applications insights
[ 2022-04-27T11:22:24     INFO ] [ exists                   ] True
[ 2022-04-27T11:22:25     INFO ] [ get                      ] {'additional_properties': {}, 'id': '/subscriptions/b698965a-1f53-45ec-8b81-58458d1b25f7/resourceGroups/VMSSReImageRun1SOUTHCENTRALUS', 'name': 'VMSSReImageRun1SOUTHCENTRALUS', 'type': 'Microsoft.Resources/resourceGroups', 'properties': <azure.mgmt.resource.resources.v2021_04_01.models._models_py3.ResourceGroupProperties object at 0x7fb2b1fd04c0>, 'location': 'southcentralus', 'managed_by': None, 'tags': None}
[ 2022-04-27T11:22:27     INFO ] [ create                   ] {'additional_properties': {}, 'id': '/subscriptions/b698965a-1f53-45ec-8b81-58458d1b25f7/resourceGroups/VMSSReImageRun1SOUTHCENTRALUS/providers/Microsoft.Storage/storageAccounts/vmsspipeline9fc2a6bc', 'name': 'vmsspipeline9fc2a6bc', 'type': 'Microsoft.Storage/storageAccounts', 'tags': {'confidentiality': 'public'}, 'location': 'southcentralus', 'sku': <azure.mgmt.storage.v2021_09_01.models._models_py3.Sku object at 0x7fb2b1e072e0>, 'kind': 'StorageV2', 'identity': None, 'extended_location': None, 'provisioning_state': 'Succeeded', 'primary_endpoints': <azure.mgmt.storage.v2021_09_01.models._models_py3.Endpoints object at 0x7fb2b1e073a0>, 'primary_location': 'southcentralus', 'status_of_primary': 'available', 'last_geo_failover_time': None, 'secondary_location': None, 'status_of_secondary': None, 'creation_time': datetime.datetime(2022, 4, 26, 16, 3, 34, 184382, tzinfo=<isodate.tzinfo.Utc object at 0x7fb2b5b0e610>), 'custom_domain': None, 'sas_policy': None, 'key_policy': None, 'key_creation_time': <azure.mgmt.storage.v2021_09_01.models._models_py3.KeyCreationTime object at 0x7fb2b1e070a0>, 'secondary_endpoints': None, 'encryption': <azure.mgmt.storage.v2021_09_01.models._models_py3.Encryption object at 0x7fb2b1e07760>, 'access_tier': 'Hot', 'azure_files_identity_based_authentication': None, 'enable_https_traffic_only': True, 'network_rule_set': <azure.mgmt.storage.v2021_09_01.models._models_py3.NetworkRuleSet object at 0x7fb2b1e07340>, 'is_sftp_enabled': None, 'is_local_user_enabled': None, 'is_hns_enabled': None, 'geo_replication_stats': None, 'failover_in_progress': None, 'large_file_shares_state': None, 'private_endpoint_connections': [], 'routing_preference': None, 'blob_restore_status': None, 'allow_blob_public_access': True, 'minimum_tls_version': 'TLS1_0', 'allow_shared_key_access': None, 'enable_nfs_v3': None, 'allow_cross_tenant_replication': None, 'default_to_o_auth_authentication': None, 'public_network_access': None, 'immutable_storage_with_versioning': None, 'allowed_copy_scope': None, 'storage_account_sku_conversion_status': None, 'dns_endpoint_type': None}
[ 2022-04-27T11:22:28     INFO ] [ get                      ] {'additional_properties': {}, 'id': '/subscriptions/b698965a-1f53-45ec-8b81-58458d1b25f7/resourceGroups/VMSSReImageRun1SOUTHCENTRALUS/providers/Microsoft.Network/networkSecurityGroups/vmssPeregrineTest-nsg', 'name': 'vmssPeregrineTest-nsg', 'type': 'Microsoft.Network/networkSecurityGroups', 'location': 'southcentralus', 'tags': {'confidentiality': 'public'}, 'etag': 'W/"032dcc53-bfb9-4cda-95f8-9d8fe45e7237"', 'security_rules': [<azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7fb2b174a220>], 'default_security_rules': [<azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7fb2b174a100>, <azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7fb2b174a340>, <azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7fb2b174a370>, <azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7fb2b174a3a0>, <azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7fb2b174a3d0>, <azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7fb2b174a400>], 'network_interfaces': None, 'subnets': [<azure.mgmt.network.v2021_05_01.models._models_py3.Subnet object at 0x7fb2b174a460>], 'flow_logs': None, 'resource_guid': '203bc10e-1f08-45fe-8574-0dda31091c3d', 'provisioning_state': 'Succeeded'}
[ 2022-04-27T11:22:28     INFO ] [ get                      ] {'additional_properties': {}, 'id': '/subscriptions/b698965a-1f53-45ec-8b81-58458d1b25f7/resourceGroups/VMSSReImageRun1SOUTHCENTRALUS/providers/Microsoft.Network/virtualNetworks/vmssPeregrineTest-vnet', 'name': 'vmssPeregrineTest-vnet', 'type': 'Microsoft.Network/virtualNetworks', 'location': 'southcentralus', 'tags': {'confidentiality': 'public'}, 'extended_location': None, 'etag': 'W/"afdc11b6-3c43-4b31-b872-581f5fe7295f"', 'address_space': <azure.mgmt.network.v2021_05_01.models._models_py3.AddressSpace object at 0x7fb2b174a5b0>, 'dhcp_options': None, 'flow_timeout_in_minutes': None, 'subnets': [<azure.mgmt.network.v2021_05_01.models._models_py3.Subnet object at 0x7fb2b1df8730>], 'virtual_network_peerings': [], 'resource_guid': 'f9c58b12-e959-47a6-90a3-d01f242a9657', 'provisioning_state': 'Succeeded', 'enable_ddos_protection': False, 'enable_vm_protection': None, 'ddos_protection_plan': None, 'bgp_communities': None, 'encryption': None, 'ip_allocations': None}
[ 2022-04-27T11:22:28     INFO ] [ subnets_list             ] <iterator object azure.core.paging.ItemPaged at 0x7fb2b1df8b50>
[ 2022-04-27T11:22:30     INFO ] [ get                      ] {'additional_properties': {}, 'id': '/subscriptions/b698965a-1f53-45ec-8b81-58458d1b25f7/resourceGroups/VMSSReImageRun1SOUTHCENTRALUS/providers/Microsoft.Compute/virtualMachineScaleSets/vmssPeregrineTest', 'name': 'vmssPeregrineTest', 'type': 'Microsoft.Compute/virtualMachineScaleSets', 'location': 'southcentralus', 'tags': {'confidentiality': 'public'}, 'sku': <azure.mgmt.compute.v2021_11_01.models._models_py3.Sku object at 0x7fb2b14adeb0>, 'plan': None, 'identity': None, 'zones': ['1'], 'extended_location': None, 'upgrade_policy': None, 'automatic_repairs_policy': None, 'virtual_machine_profile': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineScaleSetVMProfile object at 0x7fb2b14bd400>, 'provisioning_state': 'Succeeded', 'overprovision': None, 'do_not_run_extensions_on_overprovisioned_v_ms': None, 'unique_id': '8348e0b6-fe44-4f7e-81b3-a0dbaa7dfc36', 'single_placement_group': False, 'zone_balance': None, 'platform_fault_domain_count': 1, 'proximity_placement_group': None, 'host_group': None, 'additional_capabilities': None, 'scale_in_policy': None, 'orchestration_mode': 'Flexible', 'spot_restore_policy': None, 'time_created': datetime.datetime(2022, 4, 26, 16, 26, 1, 84949, tzinfo=<FixedOffset '+00:00'>)}
[ 2022-04-27T11:22:30     INFO ] [ get_vms                  ] <iterator object azure.core.paging.ItemPaged at 0x7fb2b1776370>
[ 2022-04-27T11:22:30     INFO ] [ get_vms                  ] <iterator object azure.core.paging.ItemPaged at 0x7fb2b14bda60>
[ 2022-04-27T11:22:32     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance0H1VQ8', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb2b146d190>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb2b146d370>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb2b146d220>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb2b146d4c0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb2b146d4f0>], 'patch_status': None}
[ 2022-04-27T11:22:32     INFO ] DEBUG: {'additional_properties': {}, 'code': 'ProvisioningState/succeeded', 'level': 'Info', 'display_status': 'Provisioning succeeded', 'message': None, 'time': datetime.datetime(2022, 4, 26, 16, 26, 47, 882138, tzinfo=<FixedOffset '+00:00'>)}
[ 2022-04-27T11:22:32     INFO ] DEBUG: {'additional_properties': {}, 'code': 'PowerState/running', 'level': 'Info', 'display_status': 'VM running', 'message': None, 'time': None}
[ 2022-04-27T11:22:32     INFO ] Backing off _wait_vm_state_running(...) for 0.1s ({'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance0H1VQ8', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb2b146d190>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb2b146d370>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb2b146d220>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb2b146d4c0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb2b146d4f0>], 'patch_status': None})
[ 2022-04-27T11:22:32     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance0H1VQ8', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb2b146d760>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb2b146d940>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb2b146d7f0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb2b146da90>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb2b146dac0>], 'patch_status': None}
[ 2022-04-27T11:22:32     INFO ] DEBUG: {'additional_properties': {}, 'code': 'ProvisioningState/succeeded', 'level': 'Info', 'display_status': 'Provisioning succeeded', 'message': None, 'time': datetime.datetime(2022, 4, 26, 16, 26, 47, 882138, tzinfo=<FixedOffset '+00:00'>)}
[ 2022-04-27T11:22:32     INFO ] DEBUG: {'additional_properties': {}, 'code': 'PowerState/running', 'level': 'Info', 'display_status': 'VM running', 'message': None, 'time': None}
[ 2022-04-27T11:22:32     INFO ] Backing off _wait_vm_state_running(...) for 1.9s ({'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance0H1VQ8', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb2b146d760>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb2b146d940>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb2b146d7f0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb2b146da90>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb2b146dac0>], 'patch_status': None})
[ 2022-04-27T11:22:35     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance0H1VQ8', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb2b146dbb0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb2b146dd90>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb2b146dc40>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb2b146dee0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb2b146df10>], 'patch_status': None}
[ 2022-04-27T11:22:35     INFO ] DEBUG: {'additional_properties': {}, 'code': 'ProvisioningState/succeeded', 'level': 'Info', 'display_status': 'Provisioning succeeded', 'message': None, 'time': datetime.datetime(2022, 4, 26, 16, 26, 47, 882138, tzinfo=<FixedOffset '+00:00'>)}
[ 2022-04-27T11:22:35     INFO ] DEBUG: {'additional_properties': {}, 'code': 'PowerState/running', 'level': 'Info', 'display_status': 'VM running', 'message': None, 'time': None}
[ 2022-04-27T11:22:35     INFO ] Backing off _wait_vm_state_running(...) for 4.0s ({'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance0H1VQ8', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb2b146dbb0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb2b146dd90>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb2b146dc40>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb2b146dee0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb2b146df10>], 'patch_status': None})
[ 2022-04-27T11:22:40     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance0H1VQ8', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb2b14761c0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb2b14763a0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb2b1476250>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb2b14764f0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb2b1476520>], 'patch_status': None}
[ 2022-04-27T11:22:40     INFO ] DEBUG: {'additional_properties': {}, 'code': 'ProvisioningState/succeeded', 'level': 'Info', 'display_status': 'Provisioning succeeded', 'message': None, 'time': datetime.datetime(2022, 4, 26, 16, 26, 47, 882138, tzinfo=<FixedOffset '+00:00'>)}
[ 2022-04-27T11:22:40     INFO ] DEBUG: {'additional_properties': {}, 'code': 'PowerState/running', 'level': 'Info', 'display_status': 'VM running', 'message': None, 'time': None}
[ 2022-04-27T11:22:40     INFO ] Backing off _wait_vm_state_running(...) for 3.0s ({'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance0H1VQ8', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb2b14761c0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb2b14763a0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb2b1476250>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb2b14764f0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb2b1476520>], 'patch_status': None})
[ 2022-04-27T11:22:43     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance0H1VQ8', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb2b1476790>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb2b1476970>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb2b1476820>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb2b1476ac0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb2b1476af0>], 'patch_status': None}
[ 2022-04-27T11:22:43     INFO ] DEBUG: {'additional_properties': {}, 'code': 'ProvisioningState/succeeded', 'level': 'Info', 'display_status': 'Provisioning succeeded', 'message': None, 'time': datetime.datetime(2022, 4, 26, 16, 26, 47, 882138, tzinfo=<FixedOffset '+00:00'>)}
[ 2022-04-27T11:22:43     INFO ] DEBUG: {'additional_properties': {}, 'code': 'PowerState/running', 'level': 'Info', 'display_status': 'VM running', 'message': None, 'time': None}
[ 2022-04-27T11:22:43     INFO ] Backing off _wait_vm_state_running(...) for 10.6s ({'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance0H1VQ8', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb2b1476790>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb2b1476970>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb2b1476820>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb2b1476ac0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb2b1476af0>], 'patch_status': None})
[ 2022-04-27T11:22:54     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance0H1VQ8', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb2b1467d90>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb2b1467df0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb2b1467a60>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb2b1467a90>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb2b14676d0>], 'patch_status': None}
[ 2022-04-27T11:22:54     INFO ] DEBUG: {'additional_properties': {}, 'code': 'ProvisioningState/succeeded', 'level': 'Info', 'display_status': 'Provisioning succeeded', 'message': None, 'time': datetime.datetime(2022, 4, 26, 16, 26, 47, 882138, tzinfo=<FixedOffset '+00:00'>)}
[ 2022-04-27T11:22:54     INFO ] DEBUG: {'additional_properties': {}, 'code': 'PowerState/running', 'level': 'Info', 'display_status': 'VM running', 'message': None, 'time': None}
[ 2022-04-27T11:22:54     INFO ] Backing off _wait_vm_state_running(...) for 24.5s ({'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance0H1VQ8', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb2b1467d90>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb2b1467df0>], 'extensions': None, 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb2b1467a60>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb2b1467a90>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb2b14676d0>], 'patch_status': None})
[ 2022-04-27T11:23:13     INFO ] Exit per user request
